fapolicy_analyzer/ui/profiler_page.py

```diff
@@ -195,12 +229,19 @@ def get_env_text(self) -> Optional[str]:
         return self._get_opt_text("envText")
 
     def get_entry_dict(self):
+        """ Get Profiler UI field contents and set object attributes
+        """
+        cmd = self.get_cmd_text()
+        arg = self.get_arg_text()
+        uid = self.get_uid_text()
+        pwd = self.get_pwd_text()
+        env = self.get_env_text()
         return {
-            "cmd": self.get_cmd_text(),
-            "arg": self.get_arg_text(),
-            "uid": self.get_uid_text(),
-            "pwd": self.get_pwd_text(),
-            "env": self.get_env_text(),
+            "cmd": cmd if cmd else None,
+            "arg": arg if arg else None,
+            "uid": uid if uid else None,
+            "pwd": pwd if pwd else None,
+            "env": env if env else None,
```

Comment:
Why are you making these changes?

The `get_*_text()` functions already convert to optional.

Reply:
Valid point. This is a remnant of a change that I considered where I kept the raw values of the fields in class variables, and the expanded and processed versions of the fields also in class variables. Whitespace on the end of some fields e.g. `cmd` broke the run and/or the checks, so I may or may not need to strip() the text in field prior to assignment. I need to check this on an rpm install before going back to a single line assignment.

Reply:
Ok, then it should be fine to put the `strip()` up in the `get_*_text()` functions, or even at a single point in the `_get_opt_text()` call.